### PR TITLE
Update about page

### PR DESCRIPTION
**Pull request by matthieu-lapeyre**
suggestion for the about page

@jgrizou if you can help to make it better ...

### CODE REVIEW

File: _pages/about/en.md [hunk 1]
```diff
@@ -11,50 +11,43 @@ published: true
...
 
-It targets the needs of a multidisciplinary community where researchers, teachers, artists, and robotic enthusiasts can share their work and ideas.
-Poppy tools are designed to be modular, easy to use, and easy to integrate – providing a set of building blocks that can be easily assembled and reconfigured.
-
-## Hardware overview
-
-Our major challenge is to make the robotic hardware cheap and modular so anyone can change the morphology of his/her robot to be adapted for the experience targeted.
-
-Of course, all this work is open source and distributed under Creative Commons BY-SA licenses.
+The Poppy project is born in 2012 in the [Flowers laboratory](https://flowers.inria.fr/) at [Inria Bordeaux Sud-Ouest](http://www.inria.fr/en/centre/bordeaux).
```
Comment (ghost) [inline _pages/about/en.md]: _was born_

File: _pages/about/en.md [hunk 2]
```diff
@@ -11,50 +11,43 @@ published: true
...
-### Robot simulator
+ - Engineering fields such as AI, computer science, mechanics, electronics, machine learning...
+ - Humanities such as cognitive science, psychology...
+ - Life science such as biology, biomechanics,...
+ - Community management, scientific mediation, communication...
+ - Design such as web design, object design, UX,...
+ - Art with the need of animator to create [the illusion of life](http://en.wikipedia.org/wiki/Disney_Animation:_The_Illusion_of_Life) and emotions.
 
-Poppy creatures are being integrated in the v-rep simulator with a transparent switch from simulated to physical robots through pypot.
+If you have no idea how you could help but you would want to, you are very welcome and you can **take a look at open issues on our [GitHub](https://github.com/poppy-project/) and [call for contributions](https://forum.poppy-project.org/tags/call-for-contributions)**.
```
Comment (ghost) [inline _pages/about/en.md:45]: We should link to issues here, not repo.

File: _pages/about/en.md [hunk 3]
```diff
@@ -11,50 +11,43 @@ published: true
...
 
-### Creature specific repositories
+For github ninja, you can of course create issues to notify a problem or develop new amazing features and open pull requests to integrate your idea.
 
-Each creature has a specific software repository with basic configuration files and high-level behaviors.
+## Usage rights
 
-### Web control interface
+All the technological development work made in the Poppy project is freely available under open source licenses. Only the name usage *"Poppy"* is restricted and protected as an international trademark to protect its identity.
```
Comment (ghost) [inline _pages/about/en.md:51]: Should we refer to « the Poppy project » or « Poppy Project »?

File: _pages/about/en.md (hunk 2 above)
Comment (matthieu-lapeyre) [inline _pages/about/en.md:45]: maybe but which one ? ...

File: _pages/about/en.md (hunk 3 above)
Comment (matthieu-lapeyre) [inline _pages/about/en.md:51]: The poppy project like the visual programming :)

File: _pages/about/en.md (hunk 2 above)
Comment (ghost) [inline _pages/about/en.md:45]: Ah my bad, I read the link was leading to this repository.

**Pull request merged**